### PR TITLE
Expose some of the staking miner types over metadata

### PR DESCRIPTION
**Pull request by kianenigma**
Simply adds back some of these types that have been moved from `trait Config` to `trait MinerConfig` to the metadata.

### CODE REVIEW

**Review by niklasad1 — APPROVED** [review]

**Review by sam0x17 — APPROVED** [review]

**Review by wpank — APPROVED** [review]

File: frame/election-provider-multi-phase/src/lib.rs [hunk 1]
```diff
@@ -708,6 +708,25 @@ pub mod pallet {
 		type WeightInfo: WeightInfo;
 	}
 
+	// Expose some of the miner configs over the metadata such that they can be re-implemented.
```
Comment (EmilyOstbo) [inline frame/election-provider-multi-phase/src/lib.rs:711]:
```suggestion
	// Expose miner configs over the metadata such that they can be re-implemented.
```
nit

**Review by EmilyOstbo — APPROVED** [review]
utACK

File: frame/election-provider-multi-phase/src/lib.rs (hunk 1 above)
Comment (wpank) [reply frame/election-provider-multi-phase/src/lib.rs:711]: Looks good to me 👍

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**